src/Forms/Validation.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { createContext, Component } from "react";
+import Schema from "async-validator";
+import PropTypes from "prop-types";
+
+const DEFAULT_STATE = {
+  errors: {}
+};
+
+export const ValidationContext = createContext(DEFAULT_STATE);
+
+export default class Validation extends Component {
+  state = DEFAULT_STATE;
+
+  valueOfTarget = target => {
```

Comment:
why not just call this "fieldValue" or "inputValue" or something. We should avoid using keywords in variable names. What are keywords? Things like bool / target / class / export / const / etc.

Reply:
sure